.github/workflows/validate.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint and test code
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # - name: Install shellcheck
+      #   run:  apt-get update && apt-get install shellcheck
+      - name: Lint
+        run: make lint
+  e2e-test:
+    name: E2E test
+    runs-on: ubuntu-latest
+    needs: [lint]
+    strategy: 
+      matrix:
+        k8s_version: [v1.16.0, v1.17.0, v1.18.0, v1.19.0]
+    steps:
+      - uses: actions/checkout@v2
+      - name: test
+        run: ./e2e-tests/tests.sh
+        env:
+            E2E_KUBERNETES_VERSION: ${{ matrix.k8s_version }}
+            E2E_SETUP_MINIKUBE: true
+            E2E_SETUP_KUBECTL: true
+            E2E_MINIKUBE_VERSION: 1.15.1
+            E2E_START_MINIKUBE: true
+            E2E_MINIKUBE_DRIVER: docker
+            E2E_SUDO: sudo
```

Comment:
On our migrations @paologallinaharbur and I were using a github action taking care of setting up minikube rather than the bootstrap script, which simplifies setup a lot:

https://github.com/newrelic/k8s-metadata-injection/pull/73

I'm not strongly against using the script, but I think we should follow the same approach in all pipelines

Reply:
yes it was the action I posted in the channge... using it we can forget about any minukube config and it is more gha "standard"

Reply:
I think i tried using the gh action but for some reason wasn't working very well for this case. I'll try again.
This was just to re-use the script that we already had